src/main.ts

```diff
@@ -43,6 +45,22 @@ const router = createRouter({
 			name: 'leaderboardsList',
 			component: LeaderboardsList,
 		},
+		{
+			path: '/category/:id(\\d+)',
+			name: 'categoryView',
+			component: CategoryView,
+			props: (route) => ({
+				id: Number.parseInt(route.params.id as string, 10),
+			}),
+		},
+		{
+			path: '/leaderboard/:id(\\d+)/categories/new',
```

Comment:
We used `/create` for leaderboards so either change this to `.../create` or change the LB create path to `.../new` for consistency.

Reply:
Yeah; got it. Slipped my mind. That being said; I noticed that the LB create path has "leaderboard" in the singular. I'll change it to plural, as well.